openpgp-dsm/src/lib.rs

```diff
@@ -931,8 +1042,8 @@ impl Decryptor for DsmAgent {
                     };
 
                     let agreed_tkey = cli
-                        .agree(&agree_req)
-                        .expect("failed ECDH agreement on DSM")
+                        .__agree(&agree_req, "ECDH exchange".into())
+                        .expect("ECDH exchange")
```

Comment:
I see we use `expect` at some places instead of `context` but they appear to be regular errors too. Can you correct me if I am wrong here?

Reply:
Removed all expects - trailing from a version without proper error handling. Thanks.